promotion/datastore.go

```diff
@@ -686,6 +702,21 @@ func (pg *Postgres) SaveClaimCreds(creds *ClaimCreds) error {
 	return err
 }
 
+// MarkBatchTransferSubmitted mark this batch of transfers submitted
+func (pg *Postgres) MarkBatchTransferSubmitted(ctx context.Context, batchID *uuid.UUID) error {
+	tx, err := pg.RawDB().BeginTxx(ctx, nil)
+	if err != nil {
+		return err
+	}
+	defer pg.RollbackTx(tx)
+
+	stmt := "update claim_drain set status = 'complete', completed=true, completed_at = now() where batch_id = $1"
+	if _, err := tx.Exec(stmt, batchID); err == nil {
+		return tx.Commit()
+	}
+	return nil
```

Comment:
would we want to return an error here for tx.Exec?

Reply:
https://github.com/brave-intl/bat-go/pull/973/commits/804b092ef331a7fa090c0dd7b48b391e2fb5a1fc addressed here.